src/impact-dashboard/FacilityInformation.tsx

```diff
@@ -68,71 +72,95 @@ const FormHeaderRow: React.FC<FormHeaderRowProps> = (props) => (
-    <AgeGroupRow
-      label="Facility Staff"
-      leftKey="staffCases"
-      rightKey="staffPopulation"
-      {...props}
-    />
-    {/* empty row for spacing */}
-    <FormGridRow />
-    <FormHeaderRow label="Total Population" />
-    <AgeGroupRow
-      label="Ages Unknown"
-      leftKey="ageUnknownCases"
-      rightKey="ageUnknownPopulation"
-      {...props}
-    />
-    <AgeGroupRow
-      label="Ages 0-19"
-      leftKey="age0Cases"
-      rightKey="age0Population"
-      {...props}
-    />
-    <AgeGroupRow
-      label="Ages 20-44"
-      leftKey="age20Cases"
-      rightKey="age20Population"
-      {...props}
-    />
-    <AgeGroupRow
-      label="Ages 45-54"
-      leftKey="age45Cases"
-      rightKey="age45Population"
-      {...props}
-    />
-    <AgeGroupRow
-      label="Ages 55-64"
-      leftKey="age55Cases"
-      rightKey="age55Population"
-      {...props}
-    />
-    <AgeGroupRow
-      label="Ages 65-74"
-      leftKey="age65Cases"
-      rightKey="age65Population"
-      {...props}
-    />
-    <AgeGroupRow
-      label="Ages 75-84"
-      leftKey="age75Cases"
-      rightKey="age75Population"
-      {...props}
-    />
-    <AgeGroupRow
-      label="Ages 85+"
-      leftKey="age85Cases"
-      rightKey="age85Population"
-      {...props}
-    />
-  </FormGrid>
-);
+export const AgeGroupGrid: React.FC<AgeGroupGridProps> = ({
+  collapsible = true,
+  ...props
+}) => {
+  const [collapsed, setCollapsed] = useState(collapsible);
+  const ageSpecificCaseCounts = (
+    <>
+      <AgeGroupRow
+        label="Ages 0-19"
+        leftKey="age0Cases"
+        rightKey="age0Population"
+        {...props}
+      />
+      <AgeGroupRow
+        label="Ages 20-44"
+        leftKey="age20Cases"
+        rightKey="age20Population"
+        {...props}
+      />
+      <AgeGroupRow
+        label="Ages 45-54"
+        leftKey="age45Cases"
+        rightKey="age45Population"
+        {...props}
+      />
+      <AgeGroupRow
+        label="Ages 55-64"
+        leftKey="age55Cases"
+        rightKey="age55Population"
+        {...props}
+      />
+      <AgeGroupRow
+        label="Ages 65-74"
+        leftKey="age65Cases"
+        rightKey="age65Population"
+        {...props}
+      />
+      <AgeGroupRow
+        label="Ages 75-84"
+        leftKey="age75Cases"
+        rightKey="age75Population"
+        {...props}
+      />
+      <AgeGroupRow
+        label="Ages 85+"
+        leftKey="age85Cases"
+        rightKey="age85Population"
+        {...props}
+      />
+    </>
+  );
+  return (
+    <FormGrid>
+      <FormGridRow />
+      <FormHeaderRow label="Staff Population" />
+      <AgeGroupRow
+        label="Facility Staff"
+        leftKey="staffCases"
+        rightKey="staffPopulation"
+        {...props}
+      />
+      {/* empty row for spacing */}
+      <FormGridRow />
+      <FormHeaderRow label="Total Population" />
+      <AgeGroupRow
+        label="Ages Unknown"
```

Comment:
@cawarren, pinging about the same comment we left on #305. I had proposed continuing to use this label instead of introducing a new label for the same field, but it's your call.

Reply:
I'm worried this will be confusing if someone just reads the labels on the left - I have staff, and then I have ages unknown (Is that still staff? Where do I put the incarcerated population?)

Reply:
Okay, let's go with @cawarren's original proposal in the issue, @SoumyaMahadevan

Reply:
I added an alternate on #305 in case it's preferable or easier. I think it might be a little clearer for users, but would probably be a bit more effort as well.

Reply:
For simplicity, let's just change the 'Ages Unknown' label to 'Resident population (ages unknown)'